lib/pf/Authentication/Source/EAPTLSSource.pm

```diff
@@ -0,0 +1,104 @@
+package pf::Authentication::Source::EAPTLSSource;
+
+=head1 NAME
+
+pf::Authentication::Source::EAPTLSSource add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::Authentication::Source::EAPTLSSource
+
+=cut
+
+use strict;
+use warnings;
+use Moose;
+use pf::constants;
+use pf::Authentication::constants;
+use List::Util qw(first);
+
+extends 'pf::Authentication::Source';
+
+has '+class' => (default => 'external');
```

Comment:
That's more and internal source I think.
External is for systems that are outside of the corporate environment